backend/src/Repository/OrderEntityRepository.php

```diff
@@ -67,4 +67,25 @@ public function getSpecificOrderForStore(int $id): ?array
             
             ->getOneOrNullResult();
     }
+
+    public function getOrdersByStateForAdmin(string $state): string
+    {
+        return $this->createQueryBuilder('orderEntity')
+            ->select('COUNT(orderEntity.id)')
+
+            ->andWhere('orderEntity.state = :state')
+            ->setParameter('state', $state)
+
+            ->getQuery()
+            ->getSingleScalarResult();
+    }
+
+    public function getAllOrdersCountForAdmin(): string
```

Comment:
please make sure the return result is a string, and try it in case of null

Reply:
I tested it when it is null then it will return "0".